README.md

```diff
@@ -1,12 +1,12 @@
 # ![Shaka Player](docs/shaka-player-logo.png)
 
-Shaka Player is an open-source JavaScript library for adaptive media.  It plays
+Shaka Player is an open-source JavaScript library for adaptive media.  It plays
 adaptive media formats (such as [DASH][] and [HLS][]) in a browser, without
-using plugins or Flash.  Instead, Shaka Player uses the open web standards
+using plugins or Flash.  Instead, Shaka Player uses open web standards
```

Comment:
The word "the" was correct in the phrase "uses the open web standards MSE & EME".  Please revert this change.